packages/components/src/templates/next/components/complex/DynamicDataBanner/DynamicDataBannerClient.tsx

```diff
@@ -172,15 +174,15 @@ export const DynamicDataBannerClient = ({
     )
   }
 
-  if (isLoading || data.length !== NUMBER_OF_DATA)
+  if (data.length !== NUMBER_OF_DATA)
     return <DynamicDataBannerUI data={[]} url={url} label={label} />
 
   return (
     <DynamicDataBannerUI
       title={!!title ? dynamicData[title] : undefined}
       data={data.map((singleData) => ({
         label: singleData.label,
-        value: dynamicData[singleData.key] || "-- : --",
+        value: isLoading ? undefined : dynamicData[singleData.key] || "-- : --",
```

Comment:
there's a subtle difference in behaviour now that i'm just highlighting for sanity's sake, i'm not too sure if it matters.

previously, if `isLoading`, `data === []`. however, now, it defaults to `data` (the initial one) which then gets mapped over. the difference is if `data` is allowed to be `undefined` initially, the state might be different. i didn't check/trace so dk if this is worthy of consideration but just fyi

Reply:
yup, previously there wasn't actually a loading state. The loading state is new in this PR!